google/cloud/aiplatform/models.py

```diff
@@ -2036,3 +2058,139 @@ def list(
             location=location,
             credentials=credentials,
         )
+
+    @base.optional_sync()
+    def _wait_on_export(self, operation_future, sync=True) -> None:
+        operation_future.result()
+
+    def export_model(
+        self,
+        export_format_id: str,
+        artifact_destination: Optional[str] = None,
+        image_destination: Optional[str] = None,
+        sync: bool = True,
+    ) -> None:
+        """Exports a trained, exportable Model to a location specified by the user.
+        A Model is considered to be exportable if it has at least one `supported_export_formats`.
+        Either `artifact_destination` or `image_destination` must be provided.
+
+        Usage:
+            my_model.export(
+                export_format_id='tf-saved-model'
+                artifact_destination='gs://my-bucket/models/'
+            )
+
+            or
+
+            my_model.export(
+                export_format_id='custom-model'
+                image_destination='us-central1-docker.pkg.dev/projectId/repo/image'
+            )
+
+        Args:
+            export_format_id (str):
```

Comment:
Please add the value ids currently supported.

Reply:
Usually a particular Model only has one valid export format id, think it might be confusing to the user to list all possible export IDs. It would be best if they checked `Model.supported_export_formats` first. What do you think?

Reply:
SGTM